### ISSUE TITLE
App crashes when I click on a pin in Map View

### ISSUE THREAD

**Issue by suncommon** [ushahidi/Ushahidi_iPhone#74]
When I click on any pin in Map View, app crashes. I am using an iPhone 5 with iOS 7
Just updated the app, so I do have the most recent version.

Please advise.

Thanks,

Kevin

**Comment by suncommon** [comment]
Hello. Just checking in on this...

**Comment by r0bm1lls** [comment]
Hi,

I have just set up an installation of the platform on a linux host and the app is doing the same thing for me - on all the maps I have tried. This is such a great platform I really hope this problem gets sorted somehow soon. I'm also using iPhone 5 and iPad 2 with latex iOs. 

Thanks
Rob

**Comment by r0bm1lls** [comment]
Just to be more specific: When you tap the pin the label comes up but when you tap that for more information the app crashes. 

Rob

**Comment by suncommon** [comment]
Yes, I continue to have this problem also, with both the ipad and iphone. Also, can the record max be increased from 500 to 5000. That would be a huge improvement. Thanks!

**Comment by r0bm1lls** [comment]
This is quite an important part of the app that does not work... ? does nobody else have this problem? If not what are we doing wrong?

**Comment by r0bm1lls** [comment]
This is good news. Do we wait for an app update in app store?

**Comment by suncommon** [comment]
Great news! Thanks. Was record limit also increased from 500 to 5000?

Best,

Kevin

Dictated on iPhone, so please pardon any typos.

**Comment by r0bm1lls** [comment]
HI Guys,

I'm anxiously awaiting an update with this fix on the app store. Any idea when this might happen. The app is somewhat unusable as it is, crashing every time you click a pin on the map. 

best
Rob

**Comment by r0bm1lls** [comment]
No News. No app update. I guess we will have to put the launch of our anecdotal city project on hold for some time. What a pity, so much potential.

**Comment by isaaclimdc** [comment]
I have fixed the bug, but I don't know when the Ushahidi staff will launch an update with the fixes that I made. I'll be sure to ask them! Hang in there!
